src/main/java/uk/ac/ebi/atlas/experimentpage/metadata/CellMetadataService.java

```diff
@@ -28,6 +28,10 @@ public CellMetadataService(CellMetadataDao cellMetadataDao) {
                 characteristicTypes);
     }
 
+    public Map<String, String> getMetadataValuesForGivenType(String experimentAccession, String metedataType) {
```

Comment:
**metedataType** spelling correction, I know it's very minor but helpful.

Reply:
metedataType? sorry I didn't get it :)